components/trust-manager/development/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: cert-manager
```

Comment:
I wonder how does this dependency work in the context of ArgoCD. If it depends on the namespace that is created for the cert-manager app, maybe we should add this under the cert-manager app rather than under a new app, but need someone with more ArgoCD experience to comment.

Reply:
We did not find other solution in OCP.
There is also an issue if we will include them in the same app: `cert-manager` namespace is not created by argo, it is created by the `cert-manager` controller. If we will include `trust-manager` in this app and use the `cert-manager` namespace for it, argo might not have control for this namespace.